clients/lsp-lua.el

```diff
@@ -370,6 +378,24 @@ and `../lib` ,exclude `../lib/temp`.
    ("Lua.completion.callSnippet" lsp-lua-completion-call-snippet)
    ("Lua.color.mode" lsp-lua-color-mode)))
 
+(defun lsp-lua-language-server-install (client callback error-callback update?)
+  "Download the latest version of lua-language-server and extract it to
+`lsp-lua-language-server-install-dir'."
+  (ignore client update?)
+  (let ((tmp-zip (make-temp-file "ext" nil ".zip")))
```

Comment:
I wonder if we can automate these step? I.e., if `:store-path` is empty, automatically generate temp file name with pattern of `extXYZ.zip` since the vscode code extension file is just zip file in disguise.
That can probably new helper function `lsp-vscode-download-install`

Reply:
Yes, if there is a helper function `lsp-vscode-download-install` will be very useful, and more, a helper function `'lsp-vscode-update-...` maybe also helpful, then it will be easy to download/install/update the vscode extensions.

Reply:
Current pull request won't implement these helper functions, so I resolve this comment. 
Thanks.

Reply:
We have this support already. `lsp-download-server` `:decompess :zip`